### PR TITLE
Give donate buttons slightly more horizontal space

### PR DESCRIPTION
**Pull request by Asartea**
## Because
Currently there is very little whitespace between the learn more and donate buttons, leading to a cramped feeling

## This PR
- Gives them both a bit of horizontal margin

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behavior
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc
  - `Fix` - bug fixes
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] I have verified all tests and linters pass after making these changes.

### CODE REVIEW

File: app/views/shared/_footer.html.erb [hunk 1]
```diff
@@ -8,8 +8,8 @@
         <h2 class="text-sm text-gray-500 dark:text-gray-400">The Odin Project is funded by the community. Join us in empowering learners around the globe by supporting The Odin Project!</h2>
       </div>
       <div class="text-center mt-4">
```
Comment (KevinMulhern) [inline app/views/shared/_footer.html.erb:10]: Thanks for doing this @Asartea, definitely agree on giving these more space.

It looks like the margin on the outer edges is causing a bit of a responsive bug by pushing the buttons to a new line:
Screenshot 2023-08-09 at 09 32 47


But to simplify - Tailwind has a handy `space-` utility we can put on the the parent and take care of the gap and save us from giving each child element a margin.
 
```suggestion
      <div class="mt-4 flex justify-center space-x-6">
```

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

**Comment by ManonLef** [comment]
Great call!
I do agree they look cramped. At first I saw the result by itself and thought it was a bit more than "slightly more" spacing. 
Screenshot 2023-08-09 at 10 30 02

But looking at the majority of the pages where this shows up, it looks similar to the spacing of the lesson buttons. 
Screenshot 2023-08-09 at 10 27 23

I feel they could be moved slightly closer together still since the buttons are smaller than the lesson buttons, which would make them relatively similar. But that's up for personal preference I guess.

I'm also curious if there'd be a preference to adding the margin to the buttons or use a gap to the parent instead:
Screenshot 2023-08-09 at 10 38 15

I'd love to hear @KevinMulhern 's thoughts on this.

**Comment by Asartea** [comment]
> I'm also curious if there'd be a preference to adding the margin to the buttons as you did, or use a gap to the parent instead:

As KevinMulhern correctly pointed out, Tailwind indeed has a great utility for this I forgot exists; see the updated PR.

> I feel they could be moved slightly closer together still since the buttons are smaller than the lesson buttons, which would make them relatively similar. But that's up for personal preference I guess.

I personally feel that as soon as they diverge too far from the spacing of the lesson buttons I start noticing the difference? But would love to hear a third opinion on this

> But looking at the majority of the pages where this shows up, it looks similar to the spacing of the lesson buttons.

You'd almost think that's what I based the spacing on :)

**Review by KevinMulhern — COMMENTED** [review]
I think,

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -7,9 +7,9 @@
         <h1 class="text-2xl mb-4 font-semibold text-gray-800 dark:text-gray-200">Support us!</h1>
         <h2 class="text-sm text-gray-500 dark:text-gray-400">The Odin Project is funded by the community. Join us in empowering learners around the globe by supporting The Odin Project!</h2>
       </div>
-      <div class="text-center mt-4">
-        <%= link_to 'Learn more', support_us_path, class: 'button button--secondary' %>
-        <%= link_to 'https://opencollective.com/theodinproject/donate?amount=5', class: 'button button--primary relative', target: '_blank' do %>
+      <div class="mt-4 flex justify-center space-x-6">
+        <%= link_to 'Learn more', support_us_path, class: 'button button--secondary mx-3' %>
```
[the remark this answers is not in the document]
Comment (Asartea) [reply app/views/shared/_footer.html.erb:11]: *blinks* I was clearly not entirely awake when making these changes

**Comment by ManonLef** [comment]
Looks great to me! ❤️

**Review by KevinMulhern — APPROVED** [review]
LGTM 🚀

**Pull request merged**